src/utils/canary-deployment.ts

```diff
@@ -0,0 +1,34 @@
+const RHDH_SANDBOX_UI_FQDN = 'https://sandbox.redhat.com';
+
+export const randomIntFromInterval = (min: number, max: number): number => {
+  // min and max included
+  return Math.floor(Math.random() * (max - min + 1) + min);
```

Comment:
`Math.random()` does not provide cryptographically secure random numbers FWIW. For that we should use `Crypto.getRandomValues()` - though in this context it seems the random number is just used to random select users to navigate to the new UI.

Reply:
Thanks for letting me know. But I think using `Math.random()` is enough here for our usecase.